docs/guides/technical-docs-seo-guide.rst

```diff
@@ -0,0 +1,278 @@
+and get to all your pages.
+
+Avoid orphan pages
+~~~~~~~~~~~~~~~~~~
+
+Sphinx calls pages that don't have links to them "orphans"
+and will throw a warning while building documentation that contains an orphan
+unless the warning is silenced with the :ref:`orphan directive <sphinx:metadata>`:
+
+::
+
+    $ make html
+    sphinx-build -b html -d _build/doctrees . _build/html
+    Running Sphinx v1.8.5
+    ...
+    checking consistency... /path/to/file.rst: WARNING: document isn't included in any toctree
+    done
+    ...
+    build finished with problems, 1 warning.
+
+You can make all Sphinx warnings into errors during your build process
+by setting ``SPHINXOPTS = -W --keep-going`` in your Sphinx Makefile.
+
+
+Avoid uncrawlable content
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+While typically this isn't a problem with technical documentation,
+try to avoid content that is "hidden" from search engines.
+This includes content hidden in images or videos
+which the crawler may not understand.
+For example, if you do have a video in your docs,
+make sure the rest of that page describes the content of the video.
+
+When using images, make sure to set the image alt text or set a caption on figures.
+For Sphinx, the image and figure directives support this:
+
+.. sourcecode:: rst
+
+    .. image:: your-image.png
+        :align: right
+        :alt: A description of this image
+
+    .. figure:: your-image.png
+        :align: right
+
+        A caption for this figure
+
+
+Redirects
+~~~~~~~~~
+
+Redirects tell search engines when content has moved.
+For example, if this guide moved from ``guides/technical-docs-seo-guide.html`` to ``guides/sphinx-seo-guide.html``,
+there will be a time period where search engines will still have the old URL in their index
+and will still be showing it to users.
+This is why it is important to update your own links within your docs as well as redirecting.
+If the hostname moved from docs.readthedocs.io to docs.readthedocs.org, this would be even more important!
+
+Read the Docs supports a few different kinds of :doc:`user defined redirects </user-defined-redirects>`
+that should cover all the different cases such as redirecting a certain page for all project versions,
+or redirecting one version to another.
+
+
+Canonical URLs
+~~~~~~~~~~~~~~
+
+Anytime very similar content is hosted at multiple URLs,
+it is pretty important to set a canonical URL.
+The canonical URL tells search engines where the original version
+your documentation is even if you have multiple versions on the internet
+(for example, incomplete translations or deprecated versions).
+
+Read the Docs supports :doc:`setting the canonical URL </guides/canonical>`
+if you are using a :doc:`custom domain </custom_domains>`
+under :guilabel:`Admin` > :guilabel:`Domains`
+in the Read the Docs dashboard.
+
+
+Use a robots.txt file
+~~~~~~~~~~~~~~~~~~~~~
+
+A ``robots.txt`` file is readable by crawlers
```

Comment:
We don't have a full Guide explaining how `robots.txt` works, but we could link this FAQ somewhere in this section: https://docs.readthedocs.io/en/latest/faq.html#how-can-i-avoid-search-results-having-a-deprecated-version-of-my-docs

Reply:
I considered that and I think you're right at least for now. I think it's probably better to create a guide at some point though.

Reply:
Yes. `robots.txt` should have it's own guide, I agree.

When I wrote that FAQ I just wanted the minimal configuration required to live somewhere and I think it was a mistake to make it a FAQ, even with that small content should have been a Guide :disappointed:

Reply:
We are trying to either create a new guide or spruce up one of the existing guides at least every month. That could be the next one!